samples/net/aws_iot/sample.yaml

```diff
@@ -37,4 +37,7 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp
     platform_allow:
       - nrf54l15dk/nrf54l15/cpuapp
-    extra_args: aws_iot_SHIELD="nrf7002eb_interposer_p1;nrf7002eb"
+    extra_args:
+      - aws_iot_SHIELD="nrf7002eb_interposer_p1;nrf7002eb"
```

Comment:
we are about to deprecate interposer for 54 and switch to Eb2 https://github.com/nrfconnect/sdk-nrf/pull/20274 , should we directly use that here? FYI @bama-nordic

Reply:
We have decided to keep both eb interposer and EB-II shields for 3.0. Remove eb interposer in 3.1

Reply:
Sure, we can leave the interposer in 3.0, but shouldn't we migrate all 54 combinations in twister to use Eb2 in 3.0?